refactor: enhance HyperliquidTerms and PerpAccountPanel components

Kahnchan opened the pull request:
- Adjusted image sizes in HyperliquidTerms based on media query for better responsiveness.
- Updated padding and margin properties for improved layout consistency.
- Enhanced text size and color logic in PerpAccountPanel for better visual feedback.
- Added functionality to copy user address and open external links for address details.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Copy-to-clipboard for wallet addresses with a quick “Open in Explorer” action (now supports HyperLiquid explorer and external opening).
* **Style**
  * Improved responsive layout, spacing, padding, and button sizing across slides and confirmation screen.
  * Refined typography, gaps, and alignment per screen size.
  * Enhanced checkbox sizing and divider contrast.
* **Bug Fixes**
  * MMR percentage now colorized: green at or below 50%, red above.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Kahnchan committed:
refactor: enhance HyperliquidTerms and PerpAccountPanel components

- Adjusted image sizes in HyperliquidTerms based on media query for better responsiveness.
- Updated padding and margin properties for improved layout consistency.
- Enhanced text size and color logic in PerpAccountPanel for better visual feedback.
- Added functionality to copy user address and open external links for address details.

Kahnchan committed:
Merge branch 'x' into fix/address_and_term

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/52ef29e1-6666-48df-b904-7b22138eff2a?refs=b78d533c51f177dcb35713d32c442a806ddfe575&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/52ef29e1-6666-48df-b904-7b22138eff2a/license?refs=b78d533c51f177dcb35713d32c442a806ddfe575&source=prChecksComment)

originalix commented on packages/kit/src/views/Perp/components/TradingPanel/panels/PerpAccountPanel.tsx:
```diff
@@ -202,6 +211,42 @@ function PerpAccountPanel() {
+              <SizableText
+                size="$bodySmMedium"
+                cursor="pointer"
+                onPress={() => {
+                  copyText(userAddress ?? '');
+                }}
+              >
+                {userAddress
+                  ? `${userAddress.slice(0, 6)}...${userAddress.slice(-4)}`
+                  : ''}
```
accountUtils.shortenAddress

originalix commented on packages/kit/src/views/Perp/components/TradingPanel/panels/PerpAccountPanel.tsx:
```diff
@@ -202,6 +211,42 @@ function PerpAccountPanel() {
+                iconSize="$3.5"
+                onPress={() => {
+                  if (userAddress) {
+                    const url = `https://hypurrscan.io/address/${userAddress}`;
```
- `packages/kit/src/utils/explorerUtils.ts` 里新增一个函数：openHyperLiquidExplorerUrl

- `https://hypurrscan.io/addres` 定义到 `packages/shared/src/config/appConfig.ts` 里

Kahnchan committed:
feat: add HyperLiquid explorer URL functionality

- Introduced a new utility function to open the HyperLiquid explorer URL for user addresses.
- Updated PerpAccountPanel to utilize the new function for external link navigation.
- Added the HyperLiquid explorer URL to the app configuration for easy access.

Kahnchan committed:
Merge branch 'fix/address_and_term' of https://github.com/Kahnchan/app-monorepo into fix/address_and_term

Kahnchan committed:
feat: utilize accountUtils for address formatting in PerpAccountPanel

- Integrated accountUtils to shorten user addresses for better display in the PerpAccountPanel.
- Updated the address rendering logic to enhance user experience and maintain consistency.
